samples/WebApi/README.md

```diff
@@ -0,0 +1,130 @@
+# Description
+This is a sample application, which uses CoreRT to compile a .NET Core Web API sample. 
+# Building a WebAPI app with CoreRT
+
+## Install the .NET Core SDK
+CoreRT is an AOT-optimized .NET Core runtime. If you're new to .NET Core make sure to visit the [official starting page](http://dotnet.github.io). It will guide you through installing pre-requisites and building your first app.
+If you're already familiar with .NET Core make sure you've [downloaded and installed the .NET Core 2 SDK](https://www.microsoft.com/net/download/core).
+
+## Create your app 
+
+Open a **new** shell/command prompt window and run the following commands.
```

Comment:
Why the emphasis on **new** ?

Reply:
Original motivation was to make sure that any dependencies are installed and the PATH is reloaded. Looking back - it looks kind of strange. Changed.